cli/scan.js

```diff
@@ -14,7 +14,7 @@ const args = process.argv.splice(2)
 
 try {
   assert(
-    process.argv.length > 2,
+    args.length > 0,
```

Comment:
Had you just forgotten to change this to 0 previously, maybe when you added the splice(2)?

Reply:
Looks like @jason-jackson had fixed this in a subsequent PR with some other work, but that PR hasn't been approved/merged yet.